src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditor.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+import {
+  Pane,
+  Layer,
+} from '@folio/stripes/components';
+
+class OrderTemplatesEditor extends Component {
+  render() {
+    return (
+      <Layer isOpen>
+        <Pane
+          id="order-settings-order-templates-editor"
+          defaultWidth="fill"
+          paneTitle="Order template editor"
+          dismissible
```

Comment:
dismissible without onClose? does it work well?

Reply:
no, it doesn't as it's just set up, onClose is required in this case